### ISSUE TITLE
"Sync" Button Doesn't work

### ISSUE THREAD

**Issue by MuhiminOsim** [labib666/cpps2#29]
"Sync" button doesn't work now in profile page. It shows "Request failed with status code 404" after clicking the button.

**Comment by labib666** [comment]
Yes. We recently turned it off due to some issues with our scrapper. Will be working on it soon.
